src/metabase/api/activity.clj

```diff
@@ -29,9 +29,9 @@
                   :archived
                   :collection.authority_level [:collection.name :collection_name]]
      "table"     [Table
-                  :id :name :db_id
-                  :display_name :initial_sync_status
-                  :visibility_type [:metabase_database.name :database_name]])
+                  :id :name :db_id :active
+                  :display_name [:metabase_database.initial_sync_status :initial-sync-status]
+                  :visibility_type [:metabase_database.name :database-name]])
    (let [model-symb (symbol (str/capitalize model))
```

Comment:
The output of this query matches the recent-views shape input, so we don't have to query for any of these details either. (`:active` especailly).